main.nf

```diff
@@ -163,6 +164,12 @@ if (params.bam && params.bed && params.bai && !(params.reads || params.readPaths
         .map { row -> [ row[3], row[0], row[1], row[2] ] } // get interval name, chrm, start and stop
         .combine(ch_bam_bai)
         .set {ch_bed_bam_bai}
+} else if (params.bam && !params.skip_remove_duplicates_bam && !params.bai) {
```

Comment:
I also wanted to check here in case i missed any logic for deduplicating vs. not deduplicating